src/core/state.go

```diff
@@ -224,10 +188,10 @@ type stateProgress struct {
 	// Used to count the number of currently active/pending targets
 	numActive  int64
 	numPending int64
-	numRunning int64
```

Comment:
Why did this go?

Reply:
We increment it and decrement it, but nothing ever actually read it.